torch/distributed/distributed_c10d.py

```diff
@@ -896,6 +896,24 @@ def is_ucc_available() -> bool:
     return _UCC_AVAILABLE
 
 
+def is_backend_available(backend: str) -> bool:
```

Comment:
Here,  `is_backend_available` is added specifically for checking whether the backend is available or not. Adding a backend parameter to `torch.distributed.is_available` might be another possible way to do this.